interactive-opinion/src/templates/main.html

```diff
@@ -16,7 +16,7 @@ <h1 class="email-signup-header__heading">Best of Guardian<span class="coloured-h
                 <div class="email-signup-main">
 
                     <div class="email-signup" data-component="opinion-email-interactive">
-                      <iframe src="https://www.theguardian.com/email/form/plaintone/2313" height="60px" width="100%" scrolling="no" frameborder="0" seamless
+                      <iframe src="https://www.theguardian.com/email/form/plaintone/3811" height="60px" width="100%" scrolling="no" frameborder="0" seamless
```

Comment:
@davidfurey this updates the iframe url to use the 3811... or do you want apps and web to add people to 2313 if JS doesn't run?